### PR TITLE
Log with field error instead of err

### PR DESCRIPTION
**Pull request by potuz**
**What type of PR is this?**

Small enhancement

**What does this PR do? Why is it needed?**

It changes the field logged in some initial sync warnings from "err" to "error", this reduces slightly the footprint on the journal and simplifies a bit log-parsing scripts. 

**Which issues(s) does this PR fix?**

None filed.

### CODE REVIEW

**Review by terencechain — DISMISSED** [review]

File: beacon-chain/sync/initial-sync/round_robin.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func (s *Service) processFetchedData(
 
 	// Use Batch Block Verify to process and verify batches directly.
 	if err := s.processBatchedBlocks(ctx, genesis, data.blocks, s.chain.ReceiveBlockBatch); err != nil {
-		log.WithField("err", err.Error()).Warn("Batch is not processed")
+		log.WithField("error", err.Error()).Warn("Batch is not processed")
```
Comment (prestonvanloon) [inline beacon-chain/sync/initial-sync/round_robin.go:113]:
```suggestion
		log.WithError(err).Warn("Batch is not processed")
```

File: beacon-chain/sync/initial-sync/round_robin.go [hunk 2]
```diff
@@ -125,20 +125,20 @@ func (s *Service) processFetchedDataRegSync(
 		if err := s.processBlock(ctx, genesis, blk, blockReceiver); err != nil {
 			switch {
 			case errors.Is(err, errBlockAlreadyProcessed):
-				log.WithField("err", err.Error()).Debug("Block is not processed")
+				log.WithField("error", err.Error()).Debug("Block is not processed")
```
Comment (prestonvanloon) [inline beacon-chain/sync/initial-sync/round_robin.go:128]:
```suggestion
				log.WithError(err).Debug("Block is not processed")
```

File: beacon-chain/sync/initial-sync/round_robin.go [hunk 3]
```diff
@@ -125,20 +125,20 @@ func (s *Service) processFetchedDataRegSync(
...
 				invalidBlocks++
 			case errors.Is(err, errParentDoesNotExist):
-				log.WithField("err", err.Error()).Debug("Block is not processed")
+				log.WithField("error", err.Error()).Debug("Block is not processed")
```
Comment (prestonvanloon) [inline beacon-chain/sync/initial-sync/round_robin.go:131]:
```suggestion
				log.WithError(err).Debug("Block is not processed")
```

File: beacon-chain/sync/initial-sync/round_robin.go [hunk 4]
```diff
@@ -125,20 +125,20 @@ func (s *Service) processFetchedDataRegSync(
...
 				invalidBlocks++
 			default:
-				log.WithField("err", err.Error()).Warn("Block is not processed")
+				log.WithField("error", err.Error()).Warn("Block is not processed")
```
Comment (prestonvanloon) [inline beacon-chain/sync/initial-sync/round_robin.go:134]:
```suggestion
				log.WithError(err).Warn("Block is not processed")
```

**Review by prestonvanloon — CHANGES_REQUESTED** [review]
Thanks for this PR! The log consistency is appreciated!

With Logrus, WithError is the preferred way to handle errors. It's essentially an alias for the code you've written, except for the last message which isn't an "error" type.

https://godoc.org/github.com/sirupsen/logrus#WithError

**Comment by potuz** [comment]
> Thanks for this PR! The log consistency is appreciated!
> 
> With Logrus, WithError is the preferred way to handle errors. It's essentially an alias for the code you've written, except for the last message which isn't an "error" type.
> 
> https://godoc.org/github.com/sirupsen/logrus#WithError

Oh but I didn't write any of it just batch changed "err" for "error". Sure, your changes are perfect!

**Review by prestonvanloon — APPROVED** [review]

**Review by terencechain — APPROVED** [review]

**Pull request merged**